maat-court-data-api/src/main/java/gov/uk/courtdata/model/contributions/Contributions.java

```diff
@@ -0,0 +1,50 @@
+package gov.uk.courtdata.model.contributions;
```

Comment:
I understand this is still WIP, I'm not sure if the package structure is quite right for some of the classes. Specifically I would expect classes which are only used within a specific functional domain to sit inside a sub-package of that same domain.

i.e. If the Contributions model class is only used by classes in the package `gov.uk.courtdata.contribution` (and it's sub-packages), I would expect the `Contributions` class to sit in the package  `gov.uk.courtdata.contribution.model`

This is consistent with the majority of the existing code and is IMHO a good structure as it allows us to more easily determine the scope of usages of a given class. Also it should reduce package-tangle and ensure dependancies flow vertically in a single direction (minimising inter-package dependancies).

Am more than happy to have a discussion on this and go into more detail when you're next online and take on board any downsides to this approach I may have missed.

https://stackoverflow.com/questions/15321702/what-does-package-tangle-index-data-indicate-in-sonar